src/main/autoUpdater.test.ts

```diff
@@ -1,13 +1,17 @@
 // Copyright (c) 2016-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I was having some issues that I thought were related to types failing, so TS was supposed to help.
The problem was completely different, but since I moved it to TS, I thought it was great to have it like that.